docs/userguide/expectations.md

```diff
@@ -312,6 +312,9 @@ FAILURE
 ```
 Since NULL is neither *true* nor *false*, both expectations will report failure.
 
+**Note:**
+Beware that `not_to_contain` is not a simple negation of `to_contain`: `not_to_contain` checks whether the actual and the expected set are disjoint.
```

Comment:
I don't think that note is necessary. Matcher was written to be as close in plain English as possible and I think `not_to_contain` describe its function.

Reply:
I believe `not_to_intersect` describes its function much better. A set `A` does not contain a set `B` if and only if there exists an element of `B` that is not an element of `A`; in particular the sets do not have to be disjoint. I know what I am saying, I am a mathematician. :wink:

The first line of the section says: _Expectations provide a very convenient way to perform a check on a negated matcher._ However, `not_to_contain` is not a negation of `to_contain`.  

Anyway, I removed the note. If you change your mind, it can always be included later.

Reply:
I think `not to intersect` would be a more accurate name indeed but maybe less plain. We tried to stick to plain English as possible so idea is that set A not contain set B.